packages/arb-token-bridge-ui/src/hooks/useNetworks.ts

```diff
@@ -75,10 +65,9 @@ export function isSupportedChainId(
     arbitrumGoerli.id,
     arbitrumSepolia.id,
     stylusTestnet.id,
-    xai.id,
-    xaiTestnet.id,
     arbitrumLocal.id,
     local.id,
+    ...Object.keys(orbitChains).map(Number),
```

Comment:
Related to a comment I already posted, but here we can do:
```suggestion
    ...getOrbitChains().map(chain => chain.id),
```

Reply:
done